spark/build.sbt

```diff
@@ -5,7 +5,7 @@ libraryDependencies ++= Seq(
   sparkCore % Provided,
   hadoopClient % Provided,
   "com.google.uzaygezen" % "uzaygezen-core" % "0.2",
-  "org.scalaj" %% "scalaj-http" % "2.3.0",
+  "org.scalaj" %% "scalaj-http" % "2.4.0",
```

Comment:
Makes  sense to move it into `Dependencies` file? Sounds like a good idea to move all deps there (an issue for a separate PR).

Reply:
Does this need a QC?

Also: don't we already use akka-http client somewhere?

Reply:
CQ: https://dev.eclipse.org/ipzilla/show_bug.cgi?id=16213

Nope, we only used scalaj-http; i believe akka-http was somewhere in tests.